play-json/shared/src/main/scala/play/api/libs/json/Json.scala

```diff
@@ -248,6 +248,24 @@ object Json extends JsonFacade {
    */
   def reads[A]: Reads[A] = macro JsMacroImpl.implicitConfigReadsImpl[A]
 
+  /**
+   * Creates a `Reads[A]`, if `A` is a ValueClass,
+   * by resolving at compile-time the required implicits.
```

Comment:
Can we explain what it does here? e.g. "by using the `Reads` of the underlying type". This is the behavior I expected but it might not be 100% obvious to everyone.